### PR TITLE
Update CSS scrollbar-gutter

### PR DESCRIPTION
**Pull request by fwqaaq**
#### Summary

fix: https://github.com/mdn/browser-compat-data/issues/21693

### CODE REVIEW

**Comment by queengooborg** [comment]
Hey @fwqaaq!  As the issue states, a flag must be enabled for support to be enabled, so simply setting a version number and calling the feature supported is incorrect.  You'll need to add the flag data -- please read the schema docs to see how to do this.

Also, have you confirmed the linked commit corresponds with Safari 17.1?

File: css/properties/scrollbar-gutter.json [hunk 1]
```diff
@@ -22,8 +22,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "impl_url": "https://webkit.org/b/167335"
+              "version_added": "17.1"
```
Comment (fwqaaq) [inline css/properties/scrollbar-gutter.json:25]:
```suggestion
              "version_added": "17.1",
              "flags": [
                {
                  "type": "preference",
                  "name": "#CSS-scrollerbar-gutter-property",
                  "value-to-set": true
                }
              ]
```

@queengooborg I'm not sure if what I wrote is correct, as I don't find any prompts for Safari in here: https://github.com/mdn/browser-compat-data/blob/235c0d7dcc6dd5f7d98470f28aa90cef2ce41115/schemas/compat-data-schema.md?plain=1#L418-L431

Like this: 
image

**Comment by fwqaaq** [comment]
> Also, have you confirmed the linked commit corresponds with Safari 17.1?

Yes. The version I'm using is 17.1.

File: css/properties/scrollbar-gutter.json (hunk 1 above)
Comment (queengooborg) [reply css/properties/scrollbar-gutter.json:25]: Pretty close!  We don't have to format the name, so `CSS scrollerbar-gutter property` works for the flag name!

**Comment by queengooborg** [comment]
> Yes. The version I'm using is 17.1.

What I mean is, have you cross-referenced the commit to check if the commit had landed in an earlier version, such as 17.0?

File: css/properties/scrollbar-gutter.json (hunk 1 above)
Comment (fwqaaq) [reply css/properties/scrollbar-gutter.json:25]: Oh, I see.

**Comment by fwqaaq** [comment]
> > Yes. The version I'm using is 17.1.
> 
> What I mean is, have you cross-referenced the commit to check if the commit had landed in an earlier version, such as 17.0?

image

Yeah, it is only in Safari 17.0 version, and I tested Safari 16.6 ang 17.0 version with `browserstack`, it doesn't have this property.

**Comment by queengooborg** [comment]
Oh, so it was introduced in 17.0, not 17.1!  Could you tweak the version number to say "17" instead, then?

File: css/properties/scrollbar-gutter.json [hunk 2]
```diff
@@ -22,8 +22,14 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "impl_url": "https://webkit.org/b/167335"
+              "version_added": "17.0",
```
Comment (queengooborg) [inline css/properties/scrollbar-gutter.json:25]:
```suggestion
              "version_added": "17",
```

File: css/properties/scrollbar-gutter.json [hunk 3]
```diff
@@ -22,8 +22,14 @@
...
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "impl_url": "https://webkit.org/b/167335"
+              "version_added": "17",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "CSS scrollerbar-gutter property",
+                  "value-to-set": true
```
Comment (fwqaaq) [inline css/properties/scrollbar-gutter.json:30]:
```suggestion
                  "value-to-set": "true"
```

**Review by queengooborg — APPROVED** [review]
Thank you for your PR!  Confirmed that the data is accurate per https://github.com/WebKit/WebKit/commit/85da54052e1c789e78ffae77d3dde373f2952da0, which is tagged `WebKit-7616.1.17` (corresponding to WebKit version 616.1.17), which matches Safari 17 (WebKit 616.1.27).

Welcome to BCD!  🎉   I'll merge once the above suggestion is applied which should fix the CI tests!

**Comment by fwqaaq** [comment]
Thanks for your help.

File: css/properties/scrollbar-gutter.json [hunk 4]
```diff
@@ -22,8 +22,14 @@
...
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "impl_url": "https://webkit.org/b/167335"
+              "version_added": "17",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "CSS scrollerbar-gutter property",
+                  "value-to-set": "true"
```
Comment (queengooborg) [inline css/properties/scrollbar-gutter.json:30]: One last little detail, looks like!

```suggestion
                  "value_to_set": "true"
```

Comment (fwqaaq) [reply css/properties/scrollbar-gutter.json]: Soga!

**Pull request merged**